tests/ha/single_pod_restart/test_data_pod_restart.py

```diff
@@ -1567,17 +1609,19 @@ def test_ios_during_pod_restart(self):
         LOGGER.info("Wait for %s seconds for all background operations to start",
                     HA_CFG["common_params"]["30sec_delay"])
         time.sleep(HA_CFG["common_params"]["30sec_delay"])
-        LOGGER.info("Step 7: Starting pod again by creating deployment using K8s command")
+        LOGGER.info("Step 7: Starting pod again and checking cluster status")
```

Comment:
Same as above, event_set_clr list... we will have to check this for all s3 bench background IOs.

Reply:
same reply as above